.ort.yml

```diff
@@ -31,6 +31,10 @@ excludes:
     reason: "DEV_DEPENDENCY_OF"
     comment: >-
       Packages for development only. Not part of released artifacts.
+  - pattern: "dokka.*"
+    reason: "BUILD_DEPENDENCY_OF"
+    comment: >-
+      Packages used to build artifacts. Not part of released artifacts.
```

Comment:
Consider instead: "Packages for the Dokka documentation engine." 

See also GenerateScopeExcludesCommand.kt.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/6340.